### ISSUE TITLE
Add .chezmoi.sourceFile variable to get current filename

### ISSUE THREAD

**Issue by braderhart** [twpayne/chezmoi#1210]
## Is your feature request related to a problem? Please describe.

Would like to get source filename to use in a template:

```
# This file is managed by chezmoi. Do not edit! Instead use {{ .chezmoi.sourceFile }}
```
## Describe the solution you'd like

Add a variable `.chezmoi.sourceFile` to be able to use in a template for the source file.

## Describe alternatives you've considered

Hardcode filename:

```
# This file is managed by chezmoi. Do not edit! Instead use {{ joinPath .chezmoi.sourceDir "dot_bashrc.tmpl" }}
```

**Comment by twpayne** [comment]
Thanks! Good idea :)

Implemented in #1212. Just waiting for the [GitHub Actions outage](https://www.githubstatus.com/) to resolve so the feature can be properly tested and merged.

**Comment by braderhart** [comment]
Thanks! Excited to use this!
